src/components/Input/Input.jsx

```diff
@@ -0,0 +1,54 @@
+import { forwardRef } from "react";
+import PropTypes from "prop-types";
+
+export const Input = forwardRef(
+  (
+    {
+      id,
+      label,
+      type,
+      value,
+      onChange,
+      onBlur,
+      name,
+      classes,
+      onClick,
+      isActiveClass,
+    },
+    ref,
+  ) => {
+    return (
+      <label htmlFor={id} className={`${classes} ${isActiveClass}`}>
```

Comment:
Install and use `clsx` if you still didn't. Besides, what is `isActiveClass` for? Do you need it for something specific that you didn't just ... make a style for input how it should look when `::focus`?